stdlib/join/sort_merge_join_test.go

```diff
@@ -115,7 +115,7 @@ func TestSortMergeJoinPredicateRule(t *testing.T) {
 			// compare the sort nodes created by the planner rule
 			err = plantest.ComparePlans(wantPlan, physicalPlan, func(p, q plan.Node) error {
 				ps, qs := getSortSpec(p), getSortSpec(q)
-				if (ps == nil)  && (qs == nil) {
```

Comment:
Doesn't seem relevant to the task at hand, but welcome nonetheless :rofl:

Reply:
Yes, I think that `go fmt ./...` fixed it for me, even though it wasn't part of this PR. Incidentally I don't think that our CI format check will fail a job if there are formatting inconsistencies in a test, which is how it got in there in the first place.